packages/mui-joy/src/Button/Button.test.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { describeConformance, createRenderer } from 'test/utils';
+import Button, { buttonClasses as classes } from '@mui/joy/Button';
+import { ThemeProvider } from '@mui/joy/styles';
+
+describe('Joy <Button />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Button>Conformance?</Button>, () => ({
+    render,
+    classes,
+    ThemeProvider,
+    refInstanceof: window.HTMLButtonElement,
+    muiName: 'MuiButton',
+    testVariantProps: { variant: 'contained', fullWidth: true },
+    testStateOverrides: { prop: 'size', value: 'small', styleKey: 'sizeSmall' },
+    skip: ['propsSpread', 'componentsProp', 'classesRoot'],
```

Comment:
```suggestion
    skip: ['componentsProp', 'classesRoot'],
```

Is there a valid reason for disabling this test?

Reply:
Joy Button does not inherit ButtonBase, so have to skip this test.